dpctl/tensor/_usmarray.pyx

```diff
@@ -1777,8 +1777,10 @@ cdef api object UsmNDArray_MakeSimpleFromPtr(
     Returns:
         Created usm_ndarray instance
     """
-    cdef size_t itemsize = type_bytesize(typenum)
-    cdef size_t nbytes = itemsize * nelems
+    cdef int itemsize = type_bytesize(typenum)
+    if (itemsize < 1):
+        raise ValueError("dtype with typenum=" + str(typenum) + " is not supported.")
```

Comment:
As I can see there is `test_pyx_capi_make_from_ptr` test to cover `UsmNDArray_MakeSimpleFromPtr` call. Should it be extended or the new one added to cover that negative scenario?

Reply:
I've added a test